x-pack/plugins/infra/public/plugin.ts

```diff
@@ -308,9 +310,11 @@ export class Plugin implements InfraClientPluginClass {
         const [coreStart, plugins, pluginStart] = await core.getStartServices();
         const { renderApp } = await import('./apps/metrics_app');
 
+        const isCloudEnabled = pluginsSetup.cloud?.isCloudEnabled || !!pluginsSetup.cloud?.cloudId;
```

Comment:
The `!!pluginsSetup.cloud?.cloudId` part might be redundant as it seems like `isCloudEnabled` is [directly derived](https://github.com/elastic/kibana/blob/main/x-pack/plugins/cloud/public/plugin.tsx#L58) from `cloudId`

Reply:
Thanks, yes, the `pluginsSetup.cloud?.isCloudEnabled` should be enough. Removed ✔️